frontend/src/pages/ExperimentDetails.tsx

```diff
@@ -194,7 +195,7 @@ class ExperimentDetails extends Page<{}, ExperimentDetailsState> {
                   <div className={classes(css.cardContent, !!activeRecurringRunsCount && css.recurringRunsActive)}>
                     {activeRecurringRunsCount + ' active'}
                   </div>
-                  <Button className={css.cardBtn} disableRipple={true}
+                  <Button className={css.cardBtn} id='manageExperimentRecurringRunsBtn' disableRipple={true}
```

Comment:
very nit: but can we go for shorter names using the context? Maybe "manageBtn", "expandDescriptionBtn", and "closeDialogBtn"?

Reply:
These ID's are global though, right? 

`closeDialogBtn` is very generic.

I figured this would be safer since it doesn't really matter.

I can change them if you'd prefer though

Reply:
SG then.